engine/lib/generic_printer/generic_printer.ml

```diff
@@ -477,51 +484,49 @@ module Make (F : Features.T) = struct
             self#expr'_GlobalVar_concrete ~super concrete
         | `Primitive primitive ->
             self#expr'_GlobalVar_primitive ~super primitive
+        | `TupleCons 0 -> string "tt"
```

Comment:
Same, this is Coq specific.
When do we call `_do_not_override_expr'_GlobalVar` or a `TupleCons _`?
We should fix that!

Reply:
Ah, it's the constructor for `()` that ends up as a simple `GlobalVar ()`?
If so, we should call `_do_not_override_expr'_Construct` here!

Reply:
Yes, it was a problem in some of the snapshots. I changed it to use `_do_not_override_expr'_Construct`